socialhome/streams/app/components/StampedElement.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+    <div>
+        <h2>{{ stampedContent.h2 }}</h2>
+        <p>{{ stampedContent.p }}</p>
+    </div>
+</template>
+
+<script>
+    import Vue from "vue";
```

Comment:
Do you mind if we ditch the semicolons for more Pythonic javascript? I would very much like that.

Need to make an ESlint file btw.. I put some ideas to use Airbnb style without semicolons in [architecture notes](http://socialhome.readthedocs.io/en/latest/architecture.html#code-style).